lib/core/src/server/common/babel.ts

```diff
@@ -31,12 +31,19 @@ export const plugins = [
    */
   require.resolve('@babel/plugin-proposal-optional-chaining'),
   require.resolve('@babel/plugin-proposal-nullish-coalescing-operator'),
+  [
+    require.resolve('babel-plugin-polyfill-corejs3'),
+    {
+      method: "usage-global",
+      absoluteImports: require.resolve('core-js')
```

Comment:
```suggestion
      absoluteImports: require.resolve('core-js'),
      version: require('core-js/package.json').version
```

Reply:
the plugin itself is a corejs 3 plugin, im not sure this is a thing anymore. it existed in the preset-env stuff because it tried to handle both at once, but this doesn't, we've explicitly imported the corejs3 plugin

Reply:
Yes, but different versions of the 3.x release has different polyfills

Reply:
It's still needed: this plugin outputs code that is compatible with every `core-js@3` version, so it cannot inject imports to polyfills added, for example, in `core-js@3.7` because that would break with `core-js@3.6`.

Reply:
it amazes me that core-js' minor versions are incompatible. i will add the change later today

Reply:
Not incompatible, just that newer versions have more polyfills. Without specifying a version the plugin will assume 3.0 and not polyfill newer features

Reply:
true i guess the breaking change would be on babel's end, to start pulling in the new imports. which is why we have to tell it which version, since they'd have to track the latest otherwise and enforce it

Reply:
> to start pulling in the new imports

That's why they don't, they only pull in what would be in version 3.0 if no version is specified